extensions/resteasy-reactive/quarkus-resteasy-reactive-jsonb/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -1,4 +1,3 @@
----
```

Comment:
Given `yamllint` asks for this to be present by default, I wouldn't actively remove it. I won't block this PR for this but let's not remove them in others please.

Reply:
@gsmet there are meant for multi documents yaml file which is not used here, maybe yamllint needs some setting and not the other way around..